content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -0,0 +1,244 @@
+Title: How to develop a WebApp with Kinto
+Date: 2016-12-01 10:20
+Slug: develop-a-chat-webapp-with-kinto
+Authors: Enguerran Colson
+Summary: An example of how Kinto handles a webapp's data storage.
+
```

Comment:
We should add a notice explaining that this article was first published on @enguerran's blog :)